kinds/teamrole/teamrole_kind.cue

```diff
@@ -0,0 +1,28 @@
+package kind
+
+import "time"
+
+name:        "TeamRole"
+maturity:    "merged"
+description: "An association between a Team and a Role"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// Namespace aka tenant/org id.
+				namespace: string
+
+				// Unique role uid.
+				roleUid: string
+
+				// Unique team name
+				teamName: string
```

Comment:
Not sure what the proper approach is here, in the existing table everything uses internal ids which we [don't add to schemas](https://docs.google.com/document/d/1X_R-R5LIhGoqvZUec7tQyaPXduQW_qivWJjFao5-LJ4/edit?disco=AAAAlwmuV-c)